frontend/app/forms/bank/form-list.component.js

```diff
@@ -191,6 +191,34 @@ export default class FormList extends React.Component {
                 "/static/forms/draft-court-order-motion-waive-fees/draft_court_order_motion_waive_fees-3.png"
               ]}
             />
+            <FillableForm
+              name="Petition to Expunge Records (Dismissal or Acquittal)"
```

Comment:
Are you planning no working on this form, too? The PR title and the components seem to be for the "charges never filed" form, not the dismissal form.

Reply:
This change came from rebasing to master. I didn't mean to add this change. I will fix it.